setup.py

```diff
@@ -53,13 +54,8 @@ def read_version():
         'concourse',
     ],
     packages = ['pypi_resource', 'test'],
-    install_requires = [
-        'pkginfo',
-        'twine',
-        'pep440',
-        'pip',
-    ],
-    setup_requires = [],
+    install_requires = subprocess.check_output(['pipenv', 'requirements'], text=True).splitlines()[1:], # [1:] is here beacuse we want to ignore the first line: `-i https://pypi.org/simple`
```

Comment:
Package now strictly depends on pip version to which its pinned, so when installing the package in Docker image the correct dependencies will be used. Prevents the https://github.com/cf-platform-eng/concourse-pypi-resource/issues/84